.golangci.yml

```diff
@@ -34,6 +34,8 @@ linters-settings:
       - wrapperFunc
   gocyclo:
     min-complexity: 15
+  gofmt:
```

Comment:
Why this change?

Reply:
The first commit which I made for the PR did not pass the golangci-lint check with an error msg: "File is not `gofmt`-ed with `-s` (gofmt)". Then I found out that this is an error that occurs mainly on Windows (and since I'm also on windows) with correctly formatted .go files that have CRLF line endings. To solve this I had to use the -s flag which stands for 'simplify' inside the gofmt.

Reply:
This is not an error, this means you may need to run `gofmt -s -w` on your files to fix it. We should fix the errors in files rather than amending the CI rules until and unless the rule is buggy 🙂

Reply:
Ohhh. Shall I delete that line and make the PR again after fixing it?

Reply:
Yes, please revert the change and update PR accordingly.